server/pairing/preflight/preflight.go

```diff
@@ -34,8 +34,9 @@ func preflightHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func makeCert(address net.IP) (*tls.Certificate, []byte, error) {
-	notBefore := time.Now()
-	notAfter := notBefore.Add(time.Minute)
+	now := time.Now()
+	notBefore := now.Add(-time.Minute)
```

Comment:
can we add `time.Minute` into a variable? and we can call it something like:
`certificateMaxClockDrift` or something similar?